provider/correctness_test.go

```diff
@@ -743,11 +835,12 @@ func getTrainingSetDatasetTS(storeType pt.Type, storeConfig pc.SerializedConfig)
 	}
 }
 
-func getTrainingSetFeaturesTSLabelsNoTS(storeType pt.Type, storeConfig pc.SerializedConfig) testSQLTrainingSetData {
-	db := fmt.Sprintf("DB_%s", strings.ToUpper(uuid.NewString()[:5]))
-	schema := fmt.Sprintf("SCHEMA_%s", strings.ToUpper(uuid.NewString()[:5]))
-	loc := pl.NewFullyQualifiedSQLLocation(db, schema, "TEST_FEATURES_SURF_READINGS_TABLE")
-	labelLoc := pl.NewFullyQualifiedSQLLocation(db, schema, "TEST_LABEL_LOC_LEVEL_TABLE")
+func getTrainingSetFeaturesTSLabelsNoTS(tester offlineSqlStoreTester, storeType pt.Type, storeConfig pc.SerializedConfig) testSQLTrainingSetData {
+	db := tester.GetTestDatabase()
+	locSchema := fmt.Sprintf("SCHEMA_%s", strings.ToUpper(uuid.NewString()[:5]))
```

Comment:
Why don't we have a GetTestSchema? And do the same trick rather than this stuff. Or use NewSchema on the tester?